go.mod

```diff
@@ -44,14 +44,14 @@ require (
 	github.com/gorilla/websocket v1.5.1
 	github.com/jinzhu/copier v0.3.5
 	github.com/kubernetes-csi/csi-lib-utils v0.6.1
-	github.com/longhorn/backing-image-manager v1.5.5-rc1
+	github.com/longhorn/backing-image-manager v1.5.5-rc2
```

Comment:
Should [f482ffad88fcf7f262815818d476ffde1052b979](https://github.com/longhorn/backing-image-manager/commits/v1.5.x/)?